src/odemis/gui/util/img.py

```diff
@@ -1775,13 +1775,7 @@ def draw_legend_multi_streams(images, buffer_size, buffer_scale,
         if stream is None and hasattr(s, "tint"):
             tint = s.tint.value
             
-            if isinstance(tint, colors.Colormap) or tint == FIT_TO_RGB and tint != (255, 255, 255):
-
-                # convert tint to a color map
-                if isinstance(tint, tuple):
-                    # swap r and b channels
-                    b, g, r = tuple(tint)
-                    tint = r, g, b
+            if isinstance(tint, colors.Colormap) or tint == FIT_TO_RGB or tint != RGB_AS_IS:
```

Comment:
There is something fishy here.
`... or tint != RGB_AS_IS` means that it'll always go inside, unless `tint == RGB_AS_IS`. So the previous tests are not needed. I doubt this is actually what you meant.